src/components/Notifications.js

```diff
@@ -16,14 +26,159 @@ const defaultNotificationProps = {
+      [type === 'info', colors.darkBlue],
+      [type === 'warn', colors.orange],
+      [type === 'error', colors.red],
+    ) || colors.darkBlue
+    const iconType = Utils.cond(
+      [type === 'success', 'success-standard'],
+      [type === 'warn' || type === 'error', 'warning-standard'],
+      undefined
+    )
+
+    return div({
+      style: {
+        backgroundColor: color[0],
+        borderRadius: '4px',
+        boxShadow: Style.standardShadow,
+        color: 'white',
+        cursor: 'auto',
+        display: 'flex',
+        flexDirection: 'column',
+        fontSize: 12
+      }
+    }, [
+      // content and close button
+      div({ style: { display: 'flex', padding: '0.75rem 1rem' } }, [
+        // content
+        div({ style: { display: 'flex', flex: 1, flexDirection: 'column' } }, [
+          // icon and title
+          div({ style: { alignItems: 'center', display: 'flex' } }, [
+            !!iconType && icon(iconType, { className: 'is-solid', size: 26, style: { marginRight: '0.5rem' } }),
+            div({ style: { fontWeight: 600 } }, [title])
+          ]),
+          !!message && div({ style: { marginTop: '0.5rem' } }, [message]),
+          !!detail && h(Clickable, {
+            style: { marginTop: '0.25rem', textDecoration: 'underline' },
+            onClick: () => this.setState({ modal: true })
+          }, ['Details'])
+        ]),
+        h(Clickable, {
+          title: 'Dismiss notification',
+          onClick: () => notificationsRef.current.removeNotification(id)
+        }, [icon('times', { size: 20 })])
+      ]),
+      notifications.length > 1 && div({ style: { alignItems: 'center', borderTop: `1px solid ${color[1]}`, display: 'flex', fontSize: 10, padding: '0.75rem 1rem' } }, [
+        h(Clickable, {
+          disabled: onFirst,
+          style: { color: onFirst ? color[1] : null },
+          onClick: () => this.setState({ notificationNumber: notificationNumber - 1 })
+        }, [icon('angle left', { size: 12 })]),
+        div({
+          style: {
+            backgroundColor: color[1],
+            borderRadius: 10,
+            padding: '0.2rem 0.5rem'
+          }
+        }, [
+          notificationNumber + 1, '/', notifications.length
+        ]),
+        h(Clickable, {
+          disabled: onLast,
+          style: { color: onLast ? color[1] : null },
+          onClick: () => this.setState({ notificationNumber: notificationNumber + 1 })
+        }, [icon('angle right', { size: 12 })])
+      ]),
+      modal && h(Modal, {
+        width: 800,
+        title,
+        showCancel: false,
+        showX: true,
+        onDismiss: () => this.setState({ modal: false }),
+        okButton: buttonPrimary({ onClick: () => refreshPage() }, 'Refresh Page')
+      }, [
+        h(ErrorView, { error: detail, collapses: false })
+      ])
+    ])
+  }
+})
+
+const refreshPage = () => {
+  notificationStore.set([])
+  StateHistory.clearCurrent()
+  // document.location.reload()
```

Comment:
This should probably be uncommented.

Reply:
Update: removing reset of `notificationStore`. Page reload will effectively do this.